source/overview/faq.rst

```diff
@@ -437,30 +438,30 @@ The following chart highlights the end user features of Mattermost and their sup
 High Trust Questions
 ----------------------
 
-What are the trust benefits of Mattermost compared to 3rd party SaaS systems that let customers manage their own encryption keys?
+What are the trust benefits of Mattermost compared to third-party SaaS systems that let customers manage their own encryption keys?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
-**Encryption doesn't mean a 3rd party SaaS vendor can't read your data**. A 3rd party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
+**Encryption doesn't mean a third-party SaaS vendor can't read your data**. A third party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
```

Comment:
```suggestion
**Encryption doesn't mean a third-party SaaS vendor can't read your data**. A third-party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
```

Reply:
(For consistency)

Reply:
Thanks, @amyblais. I was doing some reading around "third-party" vs. "third party" and it looks like I missed the mark on this instance and the one below. :)